tests/image/test_ms_ssim.py

```diff
@@ -0,0 +1,100 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from pytorch_msssim import msssim
+
+from tests.helpers import seed_all
+from tests.helpers.testers import BATCH_SIZE, NUM_BATCHES, MetricTester
+from torchmetrics.functional.image.ms_ssim import multiscale_ssim
+from torchmetrics.image.ms_ssim import MultiScaleSSIM
+
+seed_all(42)
+
+Input = namedtuple("Input", ["preds", "target"])
+
+_inputs = []
+for size, coef in [(128, 0.9), (128, 0.7)]:
+    preds = torch.rand(NUM_BATCHES, BATCH_SIZE, 1, size, size)
```

Comment:
lets try with a smaller batch size to see if that fixes the CPU allocation problem
```suggestion
    preds = torch.rand(NUM_BATCHES, 2, 1, size, size)
```

Reply:
Looks like this resolves our issue! Thanks a lot @SkafteNicki 🎉